apps/prairielearn/assets/scripts/lib/questionsTable.js

```diff
@@ -19,9 +19,11 @@ onDocumentReady(() => {
 
   window.sharingSetsList = function () {
     var data = $('#questionsTable').bootstrapTable('getData');
-    return _.keyBy(
+    const sharing_sets = _.keyBy(
       _.map(_.flatten(_.filter(_.map(data, (row) => row.sharing_sets))), (row) => row.name),
     );
+    sharing_sets['Public'] = 'Public';
+    return sharing_sets;
```

Comment:
This change together with the other one in this file cause the slightly strange behavior that if someone has a sharing set called 'Public', then when they select 'Public' from the drop down, then both the Publicly shared questions, and the questions in the sharing set 'Public' will be shown. 

Should we explicitly disallow someone naming a sharing set 'Public'? Or should we just leave it since no one will do this and the outcome isn't even that bad if they do anyway.

Reply:
screenshot of this strange behavior for your entertainment: 
![image](https://github.com/PrairieLearn/PrairieLearn/assets/25016959/ef4b1f79-0acc-4501-8746-e71191b65107)

Reply:
Ha, this is fine. Nit: could you fix the spacing (that is, ensure there's a space between the first and second public)?

I'd also advocate for making the green public the first one, if it's present.

Reply:
All done.